web/controllers/notes_controller.ex

```diff
@@ -43,4 +43,19 @@ defmodule Notox.NoteController do
        render(conn, :edit, note: note, changeset: changeset)
    end
   end
+
+  def show(conn, %{"id" => id}) do
```

Comment:
I should mention, that I'm not sure if you can do that, so you might have to close this and reopen it against that branch.